templates/cicd/pipeline_cfn.yml

```diff
@@ -219,10 +219,9 @@ Resources:
         - !Ref PipelineRole
 {{- range $index, $stage := .Stages}}
   {{- if $stage.TestCommands}}
-  BuildTestCommands{{$stage.Name}}:
+  BuildTestCommands-{{$stage.Name}}:
```

Comment:
Huh does the "-" work for the logical ID? I've never seen that before 🐙 

> The logical ID must be alphanumeric (A-Za-z0-9) 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html

Reply:
No, you're totally right. Switching back!